opba-embedded-starter/src/main/resources/application.yml

```diff
@@ -144,6 +144,38 @@ facade:
         cipher-algo: AES256_CBC
         key-algo: RSA
         len: 2048
+  # Datasafe security configuration
+  datasafe:
+    # Which keystore to use and how to encrypt keys in it
+    key-store:
+      type: BCFKS
+      encryption-algo: AES256_KWP
```

Comment:
Use AES-128 as default everywhere

Reply:
AES-128 was used for  "encryption-algo" property in all 3 property-files